interfaces/tests/reals/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+cmake_minimum_required(VERSION 2.8.11)
+
+set(reals_interface_h_files
+    real_threadapi.h
+    real_threadapi_renames.h
+)
+
+if(WIN32)
+set(reals_interface_h_files ${reals_interface_h_files}
+    real_interlocked_hl.h
+    real_interlocked_hl_renames.h
+    real_srw_lock.h
+    real_srw_lock_renames.h
+    real_string_utils.h
+    real_string_utils_renames.h
+    real_timer.h
+    real_timer_renames.h
+)
+else()
+endif()
+
+add_library(reals_interface INTERFACE)
```

Comment:
>reals_interface [](start = 12, length = 15)

This should be reals_pal, right?

Reply:
And the other places in this file.

---
In reply to: [439610026](https://github.com/Azure/azure-c-pal/pull/2#discussion_r439610026) [](ancestors = 439610026)

Reply:
Matt and I thought that we can make an interface target 'reals_interface' that conditionally links with 'reals_linux' or 'reals_win32' depending on the OS. The reals are exposed using the target 'azure_c_pal_reals'

---
In reply to: [439610181](https://github.com/Azure/azure-c-pal/pull/2#discussion_r439610181) [](ancestors = 439610181,439610026)